admin/proc-rhdh-deploy-aks.adoc

```diff
@@ -0,0 +1,315 @@
+  volumePermissions:
+    enabled: true
+----
+--
+
+. To install Helm Chart, run the following command:
++
+--
+[source]
+----
+helm -n <your_namespace> install -f values.yaml <your_deploy_name> openshift-helm-charts/redhat-developer-hub --version 1.1.0
+----
+--
+
+. Verify the deployment status using the following command:
++
+--
+[source,bash]
+----
+kubectl get deploy <your_deploy_name>-developer-hub -n <your_namespace>
+----
+--
+
+. Access the deployed {product-short} using the URL: `https://<app_address>`, where <app_address> is the Ingress address obtained earlier (for example, `https://108.141.70.228`).
+. To upgrade the deployment, run the following command:
++
+--
+[source]
+----
+helm -n <your_namespace> upgrade -f values.yaml <your_deploy_name> openshift-helm-charts/redhat-developer-hub --version 1.1.0
+----
+--
+
+. To delete the deployment, run the following command:
++
+--
+[source]
+----
+helm -n <your_namespace> delete <your_deploy_name>
+----
+--
+
+== Deploying the {product} on Azure Kubernetes Services (AKS) using Operator-backed deployment
+
+You can use the following procedure to deploy your {product-short} on AKS using the Operator-backed deployment.
+
+.Procedure
+
+. Obtain the {product} Operator manifest file, named `rhdh-operator-<VERSION>.yaml`, and modify the default configuration of `db-statefulset.yaml` and `deployment.yaml` by adding the following fragment:
++
+--
+[source,yaml]
+----
+securityContext:
+  fsGroup: 300
+----
+
+Following is the specified locations in the manifests:
+
+[source]
+----
+db-statefulset.yaml: | spec.template.spec
+deployment.yaml: | spec.template.spec
+----
+--
+
+. Apply the modified Operator manifest to your Kubernetes cluster:
++
+--
+[source,bash]
+----
+kubectl apply -f rhdh-operator-<VERSION>.yaml
+----
+
+[NOTE]
+====
+Execution of the previous command is cluster-scoped and requires appropriate cluster privileges.
+====
+--
+
+. Create an `ImagePull Secret` named `rhdh-pull-secret` using your Red Hat credentials to access images from the protected `registry.redhat.io`:
++
+--
+[source,bash]
+----
+kubectl -n <your_namespace> create secret docker-registry rhdh-pull-secret \
+    --docker-server=registry.redhat.io \
+    --docker-username=<redhat_user_name> \
+    --docker-password=<redhat_password> \
+    --docker-email=<email>
+----
+--
+
+. Create an Ingress manifest file, named `rhdh-ingress.yaml`, specifying your {product-short} service name as follows:
++
+--
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: rhdh-ingress
+  namespace: <your_namespace>
+spec:
+  ingressClassName: webapprouting.kubernetes.azure.com
+  rules:
+    - http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: backstage-<your-CR-name>
+                port:
+                  name: http-backend
+----
+--
+
+. To deploy the created Ingress, run the following command:
```

Comment:
If this is optional, mark as _(Optional)_

Reply:
I'm interested in understanding why you perceive this step as optional.